### ISSUE TITLE
preview not working in 'find in files'

### ISSUE THREAD

**Issue by giraud** [giraud/reasonml-idea-plugin#427]
plugin version: 0.116

### Description

when doing a find in files (ctrl+shift+f), sometimes the preview of reasonml files is not displaying anything. 
In that case, it seems there is a stacktrace in the system logs:

![image](https://github.com/giraud/reasonml-idea-plugin/assets/239612/43562c7c-3cdd-49a8-a7fd-7e7a4361cbae)

effectiveStart is negative.

ex:
```
java.lang.IllegalArgumentException: element RPsiLet:make; startOffset 3341; endOffset=3351; effectiveStart=-490; effectiveEnd=-480; elementRange=(3831,8268); element.getTextOffset()=3835
        at com.intellij.usageView.UsageInfo.<init>(UsageInfo.java:71)
        at com.intellij.usageView.UsageInfo.<init>(UsageInfo.java:122)
        at com.intellij.find.impl.FindResultUsageInfo.<init>(FindResultUsageInfo.java:48)
        at com.intellij.find.impl.FindInProjectUtil.processSomeOccurrencesInFile(FindInProjectUtil.java:303)
        at com.intellij.find.impl.FindInProjectUtil.lambda$processUsagesInFile$4(FindInProjectUtil.java:265)
        at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:941)
        at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:68)
        at com.intellij.find.impl.FindInProjectUtil.processUsagesInFile(FindInProjectUtil.java:264)
        at com.intellij.find.impl.FindInProjectTask.processFindInFilesUsagesInFile(FindInProjectTask.java:251)
        at com.intellij.find.impl.FindInProjectTask.lambda$wrapUsageProcessor$6(FindInProjectTask.java:205)
        at com.intellij.find.impl.FindInProjectTask.lambda$findUsages$4(FindInProjectTask.java:154)
        at com.intellij.openapi.roots.impl.FilesScanExecutor.lambda$doProcessOnAllThreadsInReadAction$1(FilesScanExecutor.java:99)
        at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1154)
        at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runInReadActionWithWriteActionPriority$0(ProgressIndicatorUtils.java:75)
        at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runActionAndCancelBeforeWrite(ProgressIndicatorUtils.java:158)
        at com.intellij.openapi.progress.util.ProgressIndicatorUtils.lambda$runWithWriteActionPriority$1(ProgressIndicatorUtils.java:115)
        at com.intellij.openapi.progress.ProgressManager.lambda$runProcess$0(ProgressManager.java:66)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
        at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
        at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
        at com.intellij.openapi.progress.ProgressManager.runProcess(ProgressManager.java:66)
        at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runWithWriteActionPriority(ProgressIndicatorUtils.java:112)
        at com.intellij.openapi.progress.util.ProgressIndicatorUtils.runInReadActionWithWriteActionPriority(ProgressIndicatorUtils.java:75)
        at com.intellij.openapi.roots.impl.FilesScanExecutor.lambda$doProcessOnAllThreadsInReadAction$2(FilesScanExecutor.java:98)
        at com.intellij.openapi.roots.impl.FilesScanExecutor.lambda$processOnAllThreads$3(FilesScanExecutor.java:141)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:188)
        at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
        at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
        at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
        at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
        at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:175)
        at com.intellij.openapi.roots.impl.FilesScanExecutor.lambda$runOnAllThreads$0(FilesScanExecutor.java:65)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:241)
        at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:31)
        at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:214)
        at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:212)
        at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:203)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
        at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
        at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by giraud** [comment]
the root cause is the code added in FileBase to navigate to the make() function when the file is a React component.
I remove that functionality bc I don’t know how to fix it.

**Comment by giraud** [comment]
Fixed in 0.117
